tests/unit/gaze/io/asc_test.py

```diff
@@ -192,3 +156,16 @@ def test_from_asc_raises_exception(kwargs, exception, message):
 
     msg, = excinfo.value.args
     assert msg == message
+
+
+def test_from_asc_fills_in_experiment_metadata():
+    gaze = pm.gaze.from_asc('tests/files/eyelink_monocular_example.asc', experiment=None)
```

Comment:
can you parametrize this and check also for `eyelink_monocular_2khz_example.asc`?
at least the sampling_rate would differ.